ci-operator/step-registry/firewatch/report-issues/firewatch-report-issues-commands.sh

```diff
@@ -14,3 +14,42 @@ if [ "$FIREWATCH_FAIL_WITH_TEST_FAILURES" = "true" ]; then
 fi
 
 eval "$command"
+
+cd /tmp/${BUILD_ID}
+
+cat << EOF | python
```

Comment:
I think my thoughts on this in general are:

It would probably be better for us to develop this as a firewatch feature rather than writing inline python in a bash script. I this it would be cleaner to install gitleaks into the container via the dockerfile in the firewatch repo and develop a new firewatch command that uses a version of this command (i.e. `firewatch gitleaks --somearg SOMEARG1`). I'd also like to see if we can import the gitleaks binary from another pre-built docker image (see multi-stage build files). 

My biggest concern is maintaining in-line python code in a bash script will be hard and unnecessary when we have the repository to hold this.

Reply:
That makes sense, and I agree that would be a cleaner solution. Thanks!